src/tutorials/selfhosting/selfhost-ubuntu-sources-en.md

```diff
@@ -0,0 +1,524 @@
+# Install cozy on Ubuntu 20.04 LTS focal fossa
+
+# Introduction
+
+Cozycloud provides Debian packages and [installation instructions](https://docs.cozy.io/en/tutorials/selfhosting/selfhost-debian-pkg/) for Debian 10 buster from installation packages. However, there is no published packages for Ubuntu.
+
+This documentation describes how to install cozy from sources on an Ubuntu 20.04 LTS focal fossa server. We will also see how to host multiple cozy instances on the same server, how to activate online edition for office documents and how to upgrade cozy-stack.
+
+
+# Requirements
+
+The installation procedure requires:
+
+- An Ubuntu 20.04 LTS focal fossa server
+- A domain name (mandatory to host cozy instances secured with https and accessible from internet).
+    In this docuemnt, we will use `domain.example` as an example. You will replace it with your own domain name throughout the explanation.
+    The address of your cozy instance will be `cozy.domain.example`
+- Good system administration knowledge. Despite documentation's goal is to be pretty straightforward to follow, there are some tricky and technical parts.
+
+During installation, you will also define:
+
+- a CouchDB admininstration password
+- a CouchDB database access password
+- a cozy-stack admin password
+- You will need to provide your email address for Let's Encrypt SSL certificate validation and your cozy instance creation
+
+# Couchdb
+
+Configure CouchDB package repository
+
+    sudo apt update && sudo apt install -y curl apt-transport-https gnupg
+    curl https://couchdb.apache.org/repo/keys.asc | gpg --dearmor | sudo tee /usr/share/keyrings/couchdb-archive-keyring.gpg >/dev/null 2>&1
+    echo "deb [signed-by=/usr/share/keyrings/couchdb-archive-keyring.gpg] https://apache.jfrog.io/artifactory/couchdb-deb/ $(lsb_release -sc) main" | sudo tee /etc/apt/sources.list.d/couchdb.list >/dev/null
+
```

Comment:
Does this syntax well supported by our mkdocs template? I know that sometimes our markdown is correctly displayed on github but not on docs.cozy.

Reply:
I'm happy with the visual result as tested on my laptop. or at least it's not worse than all our other documentation.
![image](https://user-images.githubusercontent.com/2768193/144563216-95f21762-5099-4b18-a7d7-8dfa1efa30d1.png)

Reply:
Good to know thanks for taking the time to make the screenshot!